modules/account-lib/src/coin/dot/utils.ts

```diff
@@ -1,10 +1,32 @@
+import { decodeAddress, encodeAddress, Keyring } from '@polkadot/keyring';
+import { decodePair } from '@polkadot/keyring/pair/decode';
+import { KeyringPair } from '@polkadot/keyring/types';
+import { EXTRINSIC_VERSION } from '@polkadot/types/extrinsic/v4/Extrinsic';
+import { hexToU8a, isHex, u8aToHex } from '@polkadot/util';
+import { base64Decode, signatureVerify } from '@polkadot/util-crypto';
+import { isValidEd25519PublicKey, isValidEd25519SecretKey } from '../../utils/crypto';
+import { UnsignedTransaction } from '@substrate/txwrapper-core';
+import { TypeRegistry } from '@substrate/txwrapper-core/lib/types';
+import { construct, createMetadata } from '@substrate/txwrapper-polkadot';
+import base32 from 'hi-base32';
+import { KeyPair } from '.';
 import { BaseUtils } from '../baseCoin';
 import { NotImplementedError } from '../baseCoin/errors';
+import { Seed } from '../baseCoin/iface';
+import { ProxyCallArgs, TransferArgs } from './iface';
+const polkaUtils = require('@polkadot/util');
+const { createTypeUnsafe } = require('@polkadot/types');
 
+const PROXY_METHOD_ARG = 2;
```

Comment:
Have a encodeAddress and decodeAddress utils should be great in order to not call the @polkadot/keyring on the platform and only use account-lib for all.